src/layout/tests.rs

```diff
@@ -3354,6 +3367,61 @@ fn interactive_resize_on_pending_unfullscreen_column() {
     check_ops(&ops);
 }
 
+#[test]
+fn move_column_to_workspace_unfocused_with_multiple_monitors() {
+    let ops = [
+        Op::AddOutput(1),
+        Op::SetWorkspaceName {
+            new_ws_name: 101,
+            ws_name: None,
+        },
+        Op::AddWindow {
+            params: TestWindowParams::new(1),
+        },
+        Op::FocusWorkspaceDown,
+        Op::SetWorkspaceName {
+            new_ws_name: 102,
+            ws_name: None,
+        },
```

Comment:
Add a window here in order for this test to catch the last error that I pointed out